config/routes.rb

```diff
@@ -223,6 +223,7 @@
 
   # Help
   get 'help' => 'help#faq', :as => 'help'
+  get 'help/:topic' => 'help#faq'
```

Comment:
Are single quotes preferred?

If so you can configure Hound by adding this to `.hound.yml`

```
ruby:
  enabled: true
  config_file: .ruby-style.yml
```

and this to `.ruby-style.yml`

```
StringLiterals:
  Enabled: true
  EnforcedStyle: single_quotes
```